### DISCUSSION TITLE
CDN `font.css` not working 🤔

### DISCUSSION THREAD

**Discussion by sebas1208** [twilio-labs/paste#3466] [Q&A] [answered]
Hello Paste team 👋🏾, we recently launched https://twilio.com/docs/twilio-cli using Paste with Next.js 🚀

We are running lighthouse audits and we stumbled with this issue:
![Screenshot 2023-08-31 at 15 44 09](https://github.com/twilio-labs/paste/assets/5571870/b4241674-be4d-4529-aca4-0c91973b4d00)

We are loading the stylesheet as you mention in the [docs](https://paste.twilio.design/introduction/for-engineers/manual-installation#out-of-the-box-themes). We tried removing it and the fonts are still loading fine because they are also defined inline in the `Head`
![image](https://github.com/twilio-labs/paste/assets/5571870/2a2d817f-611d-4166-a52f-8c92ccb822bd)

Checking the 'Coverage' tab in chrome devtools also points that the font.css is not used:
![Screenshot 2023-08-31 at 12 12 35](https://github.com/twilio-labs/paste/assets/5571870/802e1c88-116b-48da-96db-9a8a962585d9)

So I'm wondering if we can delete the `link` from the `Head` safely? Or we are missing something?

PS: this is also happening in https://paste.twilio.design/

**Comment by SiTaggart** [comment, marked as answer]
Hi @sebas1208 

Yeah, it's working as expected from our point of view. Because you're using NextJS you probably don't need to add it to the `<head />`, the only benefit is the `preconnect` which might help with first time visitors. 

We cater for so many different use cases, our docs, and the way we build is a bit of a catch all. We mostly see a lot of SPAs that aren't statically rendered or use SSR, so using the link in the head can be beneficial to preloading the fonts before the JavaScript has downloaded and parsed. We also deal with folks who don't have access to the head at all so we load the fonts for them too. It's not necessarily harmful either way, it doesn't download the font files twice, we're just trying to help optimize the time at with the browser calls or the fonts.

NextJS prerenders all the CSS in the HTML for you, so you won't get much benefit from the link in the head.

**Reply by sebas1208** [reply]
Yeah indeed the fonts are not being download twice so that's ok, but the download of the font.css itself from the cdn was blocking the page to be rendered and is not needed in our case. So thank you to clarify this @SiTaggart 👍🏾
